index.js

```diff
@@ -163,6 +165,30 @@ function createMainWindow() {
 }
 
 app.once("browser-window-created", (event, win) => {
+	// User agents are from https://developers.whatismybrowser.com/useragents/explore/
+	const originalUserAgent = win.webContents.userAgent;
+	const userAgents = {
```

Comment:
It might also be worth allowing an override in config for more flexibility :)

Reply:
see my comment below